internal/utils/convert_types.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+func ConvertNamespacedNameToStrings(toBeConverted []types.NamespacedName) []string {
+
+	convertedSlice := make([]string, 0, len(toBeConverted))
+
+	for _, nsn := range toBeConverted {
+		convertedSlice = append(convertedSlice, GetKernelModuleReadyNodeLabel(nsn.Namespace, nsn.Name))
+	}
+	return convertedSlice
+}
+
+func SetToSlice[T comparable](set sets.Set[T]) []T {
+
+	slice := make([]T, 0, set.Len())
+
+	for item := range set {
+		slice = append(slice, item)
+	}
+	return slice
+}
```

Comment:
This is already implemented in the `sets` package as `set.UnsortedList`, therefore, we can remove this function.

Reply:
File was deleted